### ISSUE TITLE
Postcode attribution

### ISSUE THREAD

**Issue by matkoniecz** [colouring-cities/colouring-britain#268]
Looking at https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/ linked from https://osdatahub.os.uk/downloads/open/CodePointOpen it seems that we need to put attribution somewhere

As far as I can see it is missing - is it somewhere?

It looks like we should add it somewhere (at map bottom? when searching? somewhere else)

Assigning @polly64 to check do we need to add it and to decide on priority (should I do this now or after mass upload tool public release?)

**Comment by matkoniecz** [comment]
Also, @polly64 - should I release postcodes now to main site (current ones are already almost certainly from CodePointOpen anyway) or wait until this is done?

Easiest solution would be to add something like ", search results are powered by Code-Point® Open, [OGL licensed](www.nationalarchives.gov.uk/doc/open-government-licence/version/3/)" or maybe similar text in the attribution at the map bottom (though it is getting quite long)

**Comment by polly64** [comment]
@matkoniecz yes let's go ahead - with  "search results are powered by Code-Point® Open, [OGL license](https://github.com/colouring-cities/colouring-britain/issues/www.nationalarchives.gov.uk/doc/open-government-licence/version/3/)"this is the link for the https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/

**Comment by polly64** [comment]
@MKIM1008 can you check this with OS and find out what the exact attribution should be thanks and then @matkoniecz can add

**Comment by matkoniecz** [comment]
![screen02](https://github.com/colouring-cities/colouring-britain/assets/899988/75556778-b742-4da4-9236-277ebe48cd44)
made a cosmetic change to work well in license list

**Comment by MKIM1008** [comment]
> @MKIM1008 can you check this with OS and find out what the exact attribution should be thanks and then @matkoniecz can add

@polly64 @matkoniecz 
I will add this agenda to the discussion list with OS and is it correct?

**Comment by matkoniecz** [comment]
>  I will add this agenda to the discussion list with OS and is it correct?

that would work well 

(not sure whether to keep it open until it is done or should it be kept closed as it almost certainly fine in the current form)

**Comment by polly64** [comment]
@MKIM1008 can you ok this for @matkoniecz this afternoon - just if you can check attribution as shown on the licence requirements

**Comment by MKIM1008** [comment]
OS OpenData is licensed under the Open Government Licence which allows you to use the OS OpenData datasets in any way and for any purpose. We simply ask that you acknowledge us with "**Contains OS data © Crown Copyright [and database right] (year)**".​

**Comment by polly64** [comment]
@matkoniecz ok
